src/components/Post/Post.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './post.scss';
+import { CommentList } from '../CommentList/CommentList';
+import { User } from '../User/User';
+import { userTypes, commentType } from '../../types';
+
+export const Post = ({
+  title,
+  body,
+  user,
+  usersComments,
+}) => (
+  <>
+    <div className="post">
+      <User {...user} />
+      <div className="post__info">
+        <p className="post__info-title">
+          {`Theme: ${title}`}
+        </p>
+        <p className="post__info-text">{body}</p>
+      </div>
+    </div>
+    <CommentList comments={usersComments} />
+  </>
+);
+
+Post.propTypes = {
+  title: PropTypes.string.isRequired,
+  body: PropTypes.string.isRequired,
+  user: userTypes,
```

Comment:
userTypes, commentType, как понять где множественное число где нет?
давай следовать одному неймингу
```suggestion
  user: userType,
```